src/main/java/com/facebook/openwifirrm/DeviceDataManager.java

```diff
@@ -455,12 +455,13 @@ public DeviceConfig getDeviceConfig(String serialNumber, String zone) {
 	}
 
 	/**
-	 * Return config (with all config layers applied) for all devices in a given
-	 * zone, or null if not present in the topology.
+	 * Return the serail number of devices in a given zone.
 	 *
-	 * @return map of serial number to computed config
+	 * @param zone the zone that will be looked up for devi es
+	 *
+	 * @return set of serial numbers
 	 */
-	public Map<String, DeviceConfig> getAllDeviceConfigs(String zone) {
+	private Set<String> getAllDevicesInZone(String zone) {
```

Comment:
Why did you split this function? `getAllDevicesInZone()` (new function) doesn't look used in this PR. If you do need this, note this function can return null which you need to handle in `getAllDeviceConfigs()` (currently would throw NPE) and mention in `@return` javadoc.

Reply:
I think I was going to use it then I took a different approach, will revert!